components/shared/package-lock.json

```diff
@@ -0,0 +1,5213 @@
+{
+	"name": "signal-provider",
+	"version": "1.0.0",
+	"lockfileVersion": 1,
+	"requires": true,
+	"dependencies": {
```

Comment:
ah cool - was there some lerna thing we were missing that generates these?

Reply:
yes running lerna install created them all